Library/Homebrew/cmd/bottle.rb

```diff
@@ -233,14 +233,14 @@ def bottle_formula(f)
           ignores << %r{#{Regexp.escape(HOMEBREW_CELLAR)}/go/[\d\.]+/libexec}
         end
 
+        relocatable = true
         if ARGV.include? "--skip-relocation"
-          relocatable = true
           skip_relocation = true
         else
-          relocatable = !keg_contains(prefix_check, keg, ignores)
-          relocatable = !keg_contains(cellar, keg, ignores) && relocatable
-          unless prefix == prefix_check
-            relocatable = !keg_contains_absolute_symlink_starting_with?(prefix, keg) && relocatable
+          relocatable = false if keg_contains(prefix_check, keg, ignores)
+          relocatable = false if keg_contains(cellar, keg, ignores)
```

Comment:
I decided that the current implementation was intentionally avoiding short circuiting so that all issues are always flagged in verbose mode so I followed suit; I think it makes sense to maintain that behavior.